bolt/include/bolt/Utils/NameResolver.h

```diff
@@ -63,8 +63,8 @@ class NameResolver {
   }
 
   // Removes a suffix from a function name.
-  static StringRef removeSuffix(StringRef Name, StringRef Suffix) {
-    const size_t Pos = Name.find(Suffix);
+  static StringRef unify(StringRef Name) {
```

Comment:
Let's use a self-descriptive name e.g. `dropNumNames`.
This would follow the convention used in BinaryFunction.h `getPrintName`.